spec/text_message_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'text_message'
+
+describe TextMessage do
+
+  describe "#send_order_text" do
+    it 'can send text message to display order information' do
+      KEYS = {account_sid:'double',auth_token:'double',phone_number:'double'}
+      text = TextMessage.new
+      order = double(:fake_order)
+      allow(order).to receive(:display_order).and_return('order details')
+      module Twilio
+      end
+      class Twilio::REST
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.